.gitattributes

```diff
@@ -1,2 +1,15 @@
 # Auto detect text files and perform LF normalization
 * text eol=lf
+
+/docs export-ignore
+/spec export-ignore
+/.editorconfig export-ignore
+/.eslintignore export-ignore
+/.eslintrc.json export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/appveyor.yml export-ignore
+/circle.yml export-ignore
+/package-lock.json export-ignore
```

Comment:
Although not used by the `npm@3` wrapped by `apm`, this should probably still be exported for when `apm` does update...